tests/Behat/features/view-a-version.feature

```diff
@@ -13,9 +13,14 @@ Feature: View a version
 
   Scenario: I can view a selected version of a record
     When I click on the first version
+    And I wait for 3 seconds until I see the "#Form_versionForm" element
     Then I should see "Welcome to my website"
 
-  Scenario: The form fields shown are readonly
+  Scenario: Shows readonly version of all core form fields
+    Given I click on "History" in the header tabs
+    Then I should see a list of versions
+
     When I click on the first version
+    And I wait for 3 seconds until I see the "#Form_versionForm" element
     Then I should see an "#Form_versionForm_Title[readonly]" element
-    And I should see an "#Form_versionForm_URLSegment[readonly]" element
+    Then I should see an "#Form_versionForm_URLSegment[readonly]" element
```

Comment:
I think `And` was better actually

Reply:
This should be available in core - I've logged it here: https://github.com/silverstripe/silverstripe-behat-extension/issues/177